package.json

```diff
@@ -125,67 +125,67 @@
     "body-parser": "^1.19.0",
     "chalk": "^2.4.2",
     "cli-table3": "^0.5.1",
-    "colors": "^1.3.3",
+    "colors": "^1.4.0",
     "cross-os": "^1.3.0",
     "distributions-poisson-quantile": "0.0.0",
-    "dotenv": "^5.0.1",
-    "express": "^4.16.3",
+    "dotenv": "^8.1.0",
+    "express": "^4.17.1",
     "fastpriorityqueue": "^0.6.3",
-    "google-protobuf": "^3.5.0",
-    "grpc": "^1.21.1",
+    "google-protobuf": "^3.10.0",
+    "grpc": "^1.24.0",
     "gulp": "^4.0.2",
     "json-stable-stringify": "^1.0.1",
     "moment": "^2.24.0",
-    "node-forge": "^0.7.6",
+    "node-forge": "^0.9.1",
     "secp256k1": "^3.7.1",
-    "semver": "^6.2.0",
+    "semver": "^6.3.0",
     "sequelize": "^4.44.2",
-    "sqlite3": "^4.0.9",
-    "swagger-ui-express": "^4.0.7",
+    "sqlite3": "^4.1.0",
+    "swagger-ui-express": "^4.1.2",
     "toml": "^3.0.0",
-    "uuid": "^3.3.2",
+    "uuid": "^3.3.3",
     "winston": "^3.2.1",
-    "yargs": "^11.1.0"
+    "yargs": "^14.2.0"
   },
   "devDependencies": {
     "@types/bluebird": "^3.5.27",
-    "@types/chai": "^4.1.7",
-    "@types/chai-as-promised": "^7.1.0",
-    "@types/chai-http": "^3.0.5",
+    "@types/chai": "^4.2.3",
+    "@types/chai-as-promised": "^7.1.2",
+    "@types/chai-http": "^4.2.0",
     "@types/distributions-poisson-quantile": "0.0.0",
-    "@types/dotenv": "^4.0.3",
-    "@types/express": "^4.16.0",
-    "@types/gulp": "^4.0.5",
-    "@types/jest": "^24.0.12",
+    "@types/dotenv": "^6.1.1",
+    "@types/express": "^4.17.1",
+    "@types/gulp": "^4.0.6",
+    "@types/jest": "^24.0.18",
     "@types/json-stable-stringify": "^1.0.32",
     "@types/mocha": "^5.2.7",
-    "@types/node": "^10.14.12",
-    "@types/node-forge": "^0.7.5",
+    "@types/node": "^12.7.12",
+    "@types/node-forge": "^0.8.6",
     "@types/secp256k1": "^3.5.0",
-    "@types/semver": "^6.0.1",
-    "@types/sequelize": "^4.28.3",
-    "@types/sinon": "^5.0.5",
+    "@types/semver": "^6.0.2",
+    "@types/sequelize": "^4.28.5",
+    "@types/sinon": "^7.5.0",
     "@types/swagger-ui-express": "^3.0.1",
     "@types/uuid": "^3.4.5",
-    "@types/yargs": "^11.1.2",
+    "@types/yargs": "^13.0.3",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-http": "^4.3.0",
-    "concurrently": "^3.6.1",
-    "conventional-changelog-cli": "^2.0.21",
-    "grpc-tools": "^1.6.0",
-    "grpc_tools_node_protoc_ts": "^2.5.4",
-    "jest": "^24.8.0",
-    "mocha": "^5.2.0",
-    "nodemon": "^1.19.1",
-    "sinon": "^7.3.2",
-    "ts-jest": "^24.0.2",
-    "ts-node": "^8.3.0",
-    "tslint": "^5.18.0",
-    "tslint-config-airbnb": "^5.11.1",
+    "concurrently": "^5.0.0",
+    "conventional-changelog-cli": "^2.0.25",
+    "grpc-tools": "^1.8.0",
```

Comment:
This related to the channel IDs not being encoded correctly, right? Damn I wasn't aware of that. I'll leave the `grpc-tools` version then.

Reply:
Yes, although come to think of it since we are no longer using `SendToRoute` I don't think this bug is actually a problem for us any more, so maybe we *should* go ahead and upgrade? This is why the simulation tests didn't fail with the newer grpc-tools version. Also assuming we don't plan to use channel ids/routes for anything in the future, which I can't think of anything that will.

Reply:
After talking with @kilrau & @erkarl we'd be OK with upgrading grpc-tools. If that makes sense to you as well, maybe put it back in this PR? Thanks.